xbmc/video/VideoDatabase.cpp

```diff
@@ -584,22 +584,35 @@ void CVideoDatabase::CreateViews()
                  "    WHERE vv.idMedia = movie.idMovie "
                  "    AND   vv.media_type = '%s' "
                  "    AND   vv.itemType = %i "
-                 "  ) AS hasVideoExtras "
+                 "  ) AS hasVideoExtras, "
+                 "  CASE "
+                 "    WHEN vv.idFile = movie.idFile AND vv.itemType = %i THEN 1 "
+                 "    ELSE 0 "
+                 "  END AS isDefaultVersion, "
```

Comment:
This column is technically not necessary and could easily be calculated in the code processing the dataset, since both idFile columns are in the select. Wouldn't make much performance difference though.

Reply:
Let's keep it for now, please. In the other commits, I use this quite often.

Reply:
Sure, no rush. To be clear I was not writing about killing it, but about calculating it in GetDetailsForMovie() where values are assigned to the new videotag.